lib/main/typescriptEditorPane.ts

```diff
@@ -47,6 +47,11 @@ export class TypescriptEditorPane {
       editor.onDidChangePath(this.checkIfTypescript),
       editor.onDidChangeGrammar(this.checkIfTypescript),
       editor.onDidDestroy(this.destroy),
+      editor.onDidSave(() => {
+        if (atom.config.get("atom-typescript.checkAllFilesOnSave")) {
```

Comment:
Perhaps it would also be a good idea to disable `geterr` on save in `typescriptBuffer` when `checkAllFilesOnSave` is enabled? Not entirely sure myself if it is a good idea or not, but I think the previous version of the code did that.

Reply:
I think doesn't need disable `geterr` when call with `{allFiles: true}` option on save. because:
- `check-all-files` are not recognized change state of text buffer when before save.
- It response faster of diagnostics messages then `check-all-files`.